### ISSUE TITLE
[New App Shortcut]: Freeview Play

### ISSUE THREAD

**Issue by j-m** [PRProd/HA-Firemote#304]
### App Name

Freeview Play

### Device Platform

Amazon Fire

### Application Details

adb_response: mAnimationIsEntrance=true      mSurface=Surface(name=)/@0xf349d60
      mSurface=Surface(name=uk.co.freeview.firetv.explore/uk.co.freeview.firetv.explore.hbbtv.HbbTvActivity)/@0xaf679e7
  mCurrentFocus=Window{977dd24 u0 uk.co.freeview.firetv.explore/uk.co.freeview.firetv.explore.hbbtv.HbbTvActivity}
  mFocusedApp=AppWindowToken{c39eb99 token=Token{a280ce0 ActivityRecord{ff65ae3 u0 uk.co.freeview.firetv.explore/.hbbtv.HbbTvActivity t2619}}}

**Comment by PRProd** [comment]
Well... that didn't work.  :man_shrugging: 

Can you do this once again, but this time provide these fields too?
  - source_list
  - app_id
  - source

**Comment by j-m** [comment]
Here's everything that command returns: 

```
source_list:
  - uk.co.freeview.firetv.explore
app_id: uk.co.freeview.firetv.explore
source: uk.co.freeview.firetv.explore
adb_response: |-
  mAnimationIsEntrance=true      mSurface=Surface(name=)/@0xa08ac82
        mSurface=Surface(name=uk.co.freeview.firetv.explore/uk.co.freeview.firetv.explore.hbbtv.HbbTvActivity)/@0x2ebcf7c
    mCurrentFocus=Window{863e2af u0 uk.co.freeview.firetv.explore/uk.co.freeview.firetv.explore.hbbtv.HbbTvActivity}
    mFocusedApp=AppWindowToken{149c868 token=Token{cb32f8b ActivityRecord{d64cd05 u0 uk.co.freeview.firetv.explore/.hbbtv.HbbTvActivity t2634}}}
hdmi_input: null
device_class: tv
entity_picture: >-
  /api/media_player_proxy/media_player.fire_tv_192_168_0_158?token=e06ecadccacb8d7067533bec3fc598143a7e8e13d47758f91fe9ab56d0593629&cache=97a0d5aa4a92f87f
friendly_name: Fire TV 192.168.0.158
supported_features: 22961
```

**Comment by PRProd** [comment]
Firemote beta version [v.3.2.2b2](https://github.com/PRProd/HA-Firemote/releases/tag/v3.2.2b2) is ready to download and test.  

I sure hope it works this time, or else things could get tricky.

**Comment by PRProd** [comment]
Firemote beta version [v.3.2.2b3](https://github.com/PRProd/HA-Firemote/releases/tag/v3.2.2b3) is ready to download and test.

If this doesn't work, I have one more thing to try: 
Follow the directions that you find [here](https://github.com/PRProd/HA-Firemote/wiki/Getting-Android-Device-Attributes#getting-device-input-details), but use a different command.  Do **not** use adb shell dumpsys input, use this one instead:
```
adb shell pm dump uk.co.freeview.firetv.explore
```

**Comment by j-m** [comment]
b3 works like a charm!

**Comment by PRProd** [comment]
Fixed in #303

**Comment by PRProd** [comment]
Published in Firemote version [v3.2.2](https://github.com/PRProd/HA-Firemote/releases/tag/v3.2.2)
